src/proc_image_processing/cpu/algorithm/rot_rect.cc

```diff
@@ -45,7 +45,7 @@ namespace proc_image_processing {
         angle = out_angle;
     }
 
-    void RotRect::swap(RotRect &a) {
+    void RotRect::swap(RotRect &a) noexcept {
```

Comment:
first time I see this keyword..

Reply:
Code smell (mais supposément qu'on doit définir ça explicitement si ça lance pas d'exceptions)

Reply:
ok.. good to know I guess